app/src/features/surveys/view/SurveySummaryResults.tsx

```diff
@@ -55,33 +83,77 @@ export enum ClassGrouping {
 }
 
 const SurveySummaryResults = () => {
+  const BorderLinearProgress = withStyles((theme: Theme) =>
+    createStyles({
+      root: {
+        height: 6,
+        borderRadius: 3
+      },
+      colorPrimary: {
+        backgroundColor: theme.palette.grey[theme.palette.type === 'light' ? 300 : 700]
+      },
+      bar: {
+        borderRadius: 3,
+        backgroundColor: '#1976D2'
+      }
+    })
+  )(LinearProgress);
   const biohubApi = useBiohubApi();
-  const urlParams = useParams();
-  const dialogContext = useContext(DialogContext);
   const surveyContext = useContext(SurveyContext);
+  const dialogContext = useContext(DialogContext);
+
+  const projectId = surveyContext.projectId as number;
+  const surveyId = surveyContext.surveyId as number;
 
   const classes = useStyles();
 
-  const projectId = urlParams['id'];
-  const surveyId = urlParams['survey_id'];
   const [openImportSummaryResults, setOpenImportSummaryResults] = useState(false);
+  const [refreshData, setRefreshData] = useState(false);
+
+  // provide file name for 'loading' ui before submission responds
+  const [filName, setFileName] = useState('');
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  //Summary Data Loader and Error Handling
+  const summaryDataLoader = useDataLoader(() => biohubApi.survey.getSurveySummarySubmission(projectId, surveyId));
+  useDataLoaderError(summaryDataLoader, () => {
+    return {
+      dialogTitle: 'Error Loading Summary Details',
+      dialogText:
+        'An error has occurred while attempting to load summary details, please try again. If the error persists, please contact your system administrator.'
+    };
+  });
+
+  useEffect(() => {
+    summaryDataLoader.load();
```

Comment:
Could remove this from a useEffect and just call `summaryDataLoader.load()`, since the dataloader itself already has a similar `useEffect` hook internally

Reply:
removed, this could also be changing with the up coming render ticket